src/main/resources/i18n/messages.properties

```diff
@@ -530,6 +534,7 @@ settings.account.steamAccount.prompt=You can link your steam account here.
 settings.account.steamAccount.linkToSteam=Link to steam
 settings.account.username=FAF Username
 settings.account.username.change=Change username
+settings.account.username.error=Your username could not be changed. Please be aware that you can only change it every 30 days. Server reports: {0}
```

Comment:
It's nice to give the user more info, however, the 30 days are server-side configuration and might change; I'd rely on the server message instead (which should contain the explanation what failed).